recipes/confection/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "confection" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5ff894a2bddd418881381612b2581e25edba22d921fa2fa66dc9b5082acf8ad3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pydantic >=1.7.4,!=1.8,!=1.8.1,<1.10.0
+    - typing_extensions >=3.7.4.1,<4.2.0  # [py<=37]
+    - srsly >=2.4.0,<3.0.0
+
+test:
+  imports:
+    - {{ name }}
+  requires:
+    - pytest
+    - numpy
+  commands:
+    - python -m pytest --tb=native --pyargs {{ name }}
+
+
+about:
+  home: https://github.com/explosion/confection
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: The sweetest config system for Python
+  description: |
+      A lightweight library that offers a configuration system letting you
+      conveniently describe arbitrary trees of objects.
+  doc_url: https://github.com/explosion/confection
+  dev_url: https://github.com/explosion/confection
+
+extra:
+  recipe-maintainers:
```

Comment:
I'm happy to merge this once all of the co-maintainers give their ok, or you let me know that they've given permission outside of this PR.

Reply:
Thanks! I can confirm that it is fine with them. You can see many related packages from our organization with the same maintainers, e.g. `spacy` and `thinc`.